forum/serializers/contents.py

```diff
@@ -125,11 +126,11 @@ class UserContentSerializer(serializers.Serializer[dict[str, Any]]):
     course_id = serializers.CharField()
     anonymous = serializers.BooleanField(default=False)
     anonymous_to_peers = serializers.BooleanField(default=False)
-    created_at = serializers.DateTimeField()
-    updated_at = serializers.DateTimeField()
+    created_at = CustomDateTimeField()
```

Comment:
`created_at = serializers.DateTimeField(format="%Y-%m-%dT%H:%M:%SZ")`
While saving the date, we can use datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
we will not need to make CustomDateTimeField for this then.

Reply:
There are multiple date fields in the different models therefore I've created a separate PR. Moreover, %S doesn't format the date as in the v1 requests. We have to truncate it to 2 decimal places.